server/liveblog/blogs/tasks.py

```diff
@@ -21,11 +21,12 @@
 logger = logging.getLogger('superdesk')
 
 
-def publish_embed(blog_id, api_host=None, theme=None):
+def publish_embed(blog_id, theme=None, output=None, api_host=None):
```

Comment:
I think it might make sense to create separate methods instead trying to handle 2 or 3 cases in one method. So you could create a publish_output_embed method.